QUARKUS-5613.md

```diff
@@ -0,0 +1,85 @@
+## Links to the resources
+- JIRA issue: https://issues.redhat.com/browse/QUARKUS-5613
+- GH PRs related to the Jira issue: https://github.com/quarkusio/quarkus/pull/43283 https://github.com/quarkusio/quarkus/issues/42117 
+- Guides modified and related:
+  https://quarkus.io/guides/security-openid-connect-client-reference
+  https://quarkus.io/guides/security-oidc-code-flow-authentication
+  https://quarkus.io/guides/security-openid-connect-client-registration
+- TP/TD PRs:
+
+## Scope of the testing
+
+Our main testing scope is to ensure that the new response filter mechanism works as intended for various OIDC endpoints, ensuring that is possible intercept and handle responses
+from various OIDC endpoints.
+
+This complements the already available OidcRequestFilter functionality.
+
+Integration tests will be on the next module in our Test suite:    
```

Comment:
What do you mean by the `on the next module`? You named explicitly which module, are they going to be in same module, different module, or module sorted next to the one where request filters are?

I suspect it is just a typo.